### PR TITLE
Add readSVGFile code to git

### PR DESCRIPTION
**Pull request by praveena-pappu**

### CODE REVIEW

File: viz/readSVGFile/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Data Studio Community Visualization code to create SVG Images
+
+Data Studio [community visualizations][community viz] allow you to write custom
```
Comment (y3l2n) [inline viz/readSVGFile/README.md]: same comment as other PR: please use reference links (add the references at the bottom) for all links

File: viz/readSVGFile/src/index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+const dscc = require('@google/dscc');
```
Comment (y3l2n) [inline viz/readSVGFile/src/index.js]: Did you run prettier on this?

File: viz/readSVGFile/src/manifest.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Draw SVG",
+  "organization": "Data Studio Developer Relations",
+  "description": "This visualization allows user to give SVG XML as a text input and create report with image.",
```
Comment (y3l2n) [inline viz/readSVGFile/src/manifest.json]: "Use SVG XML text input to render an image."

File: viz/readSVGFile/src/manifest.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  "logoUrl": "https://storage.googleapis.com/lego-devrel-testing/readSVGFile/svg.jpg",
+  "supportUrl": "https://issuetracker.google.com//issues/new?component=801069&template=1385925",
+  "privacyPolicyUrl": "",
+  "termsOfServiceUrl": "",
+  "packageUrl": "https://datastudio.google.com/open/15pJmbAJMCyPBFWYICz_XQlU_nKjSjwqD?usp=sharing",
+  "organizationUrl":"",
+  "devMode": true,
```
Comment (y3l2n) [inline viz/readSVGFile/src/manifest.json]: devMode should be false

File: viz/readSVGFile/src/manifest.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  "components": [
+    {
+      "id":"svgFile",
+      "name": "Draw SVG",
+      "description": "User can create SVG image by providing XML format in text input field in Style tab.",
```
Comment (y3l2n) [inline viz/readSVGFile/src/manifest.json]: same comment as package description

File: viz/readSVGFile/src/manifest.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "Draw SVG",
+  "organization": "Data Studio Developer Relations",
+  "description": "This visualization allows user to give SVG XML as a text input and create report with image.",
+  "logoUrl": "https://storage.googleapis.com/lego-devrel-testing/readSVGFile/svg.jpg",
+  "supportUrl": "https://issuetracker.google.com//issues/new?component=801069&template=1385925",
+  "privacyPolicyUrl": "",
+  "termsOfServiceUrl": "",
+  "packageUrl": "https://datastudio.google.com/open/15pJmbAJMCyPBFWYICz_XQlU_nKjSjwqD?usp=sharing",
```
Comment (y3l2n) [inline viz/readSVGFile/src/manifest.json]: this report should be clearly documented and publicly sharable

File: viz/readSVGFile/README.md (hunk 1 above)
Comment (praveena-pappu) [reply viz/readSVGFile/README.md]: Updated

File: viz/readSVGFile/src/manifest.json (hunk 3 above)
Comment (praveena-pappu) [reply viz/readSVGFile/src/manifest.json]: Updated

File: viz/readSVGFile/src/manifest.json (hunk 5 above)
Comment (praveena-pappu) [reply viz/readSVGFile/src/manifest.json]: Updated

File: viz/readSVGFile/src/manifest.json (hunk 4 above)
Comment (praveena-pappu) [reply viz/readSVGFile/src/manifest.json]: Updated

File: viz/readSVGFile/src/manifest.json (hunk 6 above)
Comment (praveena-pappu) [reply viz/readSVGFile/src/manifest.json]: Updated

File: viz/readSVGFile/src/index.js (hunk 2 above)
Comment (praveena-pappu) [reply viz/readSVGFile/src/index.js]: Hi Yulan. I ran prettier while submission. It didn't allow me to push code until I ran "npm run prettier" command

**Pull request merged**